src/redux/reducers/wallet.js

```diff
@@ -1,22 +1,35 @@
-import ACTIONS from '../actionTypes'
+import ACTIONS from 'redux/actionTypes'
 
 const initState = {
-  daiBalance: '',
-  xdaiBalance: '',
```

Comment:
hm... I think this is still being used by the actions that update balance

Reply:
they're back